src/OrchardCore.Modules/OrchardCore.Media.Azure/Services/ImageSharpImageCacheOptionsConfiguration.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using OrchardCore.Environment.Shell;
+using OrchardCore.Environment.Shell.Configuration;
+using OrchardCore.Media.Azure.Helpers;
+
+namespace OrchardCore.Media.Azure.Services;
+
+internal class ImageSharpImageCacheOptionsConfiguration : IConfigureOptions<ImageSharpImageCacheOptions>
+{
+    private readonly IShellConfiguration _shellConfiguration;
+    private readonly ShellSettings _shellSettings;
+    private readonly ILogger _logger;
+    private readonly FluidParserHelper<ImageSharpImageCacheOptions> _fluidParserHelper;
+
+    public ImageSharpImageCacheOptionsConfiguration(
+        IShellConfiguration shellConfiguration,
+        ShellSettings shellSettings,
+        ILogger<ImageSharpImageCacheOptionsConfiguration> logger)
+    {
+        _shellConfiguration = shellConfiguration;
+        _shellSettings = shellSettings;
+        _logger = logger;
+        _fluidParserHelper = new(shellSettings);
+    }
+
+    public void Configure(ImageSharpImageCacheOptions options)
+    {
+        var section = _shellConfiguration.GetSection("OrchardCore_Media_Azure_ImageSharp_Cache");
+
+        options.ContainerName = section.GetValue(nameof(options.ContainerName), string.Empty);
```

Comment:
perhaps you can call here `section.Bind(options)` instead. you could still keep 

```
options.CreateContainer = section.GetValue(nameof(options.CreateContainer), true);
```

Reply:
Yeah, and that's not needed either with a default property value.